### ISSUE TITLE
lein dev fails with the newest template version (1.0.4)

### ISSUE THREAD

**Issue by kupcimat** [Day8/re-frame-template#92]
After creating a new project using the newest version of the template (1.0.4), the `lein dev` fails on invalid `shadow-cljs` config:
```
invalid config
-- Spec failed --------------------

  {:cache-root ...,
   :builds {:npm ..., :app ..., :devtools {:http-root "resources/public", :http-port 8280, :build-id :devtools}},
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   :lein ...,
   :nrepl ...,
   :browser-test ...,
   :karma-test ...,
   :user-config ...}

should contain key: :target

|     key |                                          spec |
|---------+-----------------------------------------------|
| :target | (fn [%] (or (simple-keyword? %) (symbol? %))) |
```
The problem seems to be with incorrectly generated `shadow-cljs.edn` file which looks like has an extra `}` at the end of `:modules`:
```
:modules         {:app {:init-fn striker.core/init
                                        :preloads [devtools.preload]}}}
```
After removing it, it seems to work fine. To reproduce the issue you can run:
```
lein new re-frame project +test
yarn install
lein dev
```
Thanks

**Comment by superstructor** [comment]
Fixed in 1.0.5.

Thanks for the bug report and apologies that this slipped through testing for 1.0.4!

**Comment by kupcimat** [comment]
Thank you for a quick fix!
